### ISSUE TITLE
Uploading files finalize results in an error

### ISSUE THREAD

**Issue by mikedawson** [UstadMobile/UstadMobile#189]
Call to finalizeEntries results in an error:

```
2020-08-27 16:26:22.194 [nioEventLoopGroup-4-2] ERROR Application - Unhandled: POST - /ContainerUpload/finalizeEntries/
java.lang.IllegalStateException: Could not find the path of md5sum /pD8AgEhOl9GwFS/xd5QrA==
        at com.ustadmobile.sharedse.container.ContainerManagerExtKt$addEntriesFromConcatenatedInputStream$2.invokeSuspend(ContainerManagerExt.kt:44) ~[app-ktor-server-all.jar:na]
        at com.ustadmobile.sharedse.container.ContainerManagerExtKt$addEntriesFromConcatenatedInputStream$2.invoke(ContainerManagerExt.kt) ~[app-ktor-server-all.jar:na]
        at com.ustadmobile.core.container.ContainerManager.addEntries(ContainerManager.kt:114) ~[app-ktor-server-all.jar:na]
        at com.ustadmobile.sharedse.container.ContainerManagerExtKt.addEntriesFromConcatenatedInputStream(ContainerManagerExt.kt:27) ~[app-ktor-server-all.jar:na]
        at com.ustadmobile.lib.rest.ContainerUploadRouteKt$ContainerUpload$1$2.invokeSuspend(ContainerUploadRoute.kt:76) ~[app-ktor-server-all.jar:na]
```

**Comment by mikedawson** [comment]
The attached file caused the exception above.
[cell_phone.zip](https://github.com/UstadMobile/UstadMobile/files/5136942/cell_phone.zip)

**Comment by mikedawson** [comment]
Fixed with commit 9980162566e2716027cd79658500e1306495dc47
